### PR TITLE
move config and add ToConfig

### PR DESCRIPTION
**Pull request by jptosso**
> Thank you for contributing to Coraza WAF, your effort is greatly appreciated
> Before submitting check if what you want to add to `coraza` list meets [quality standards](https://github.com/corazawaf/coraza/blob/master/CONTRIBUTING.md#quality-standards) before sending pull request. Thanks!

**Make sure that you've checked the boxes below before you submit PR:**

- [x] My code includes positive and negative tests.
- [x] I have an appropriate description with correct grammar.
- [x] I have read [Contribution guidelines](https://github.com/corazawaf/coraza/blob/master/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/corazawaf/coraza/blob/master/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/github.com/corazawaf/coraza/v3sso/coraza-waf/blob/master/CONTRIBUTING.md#quality-standards).
- [x] My code is properly linted and passes pre-commit tests.

Thanks for your contribution :heart:

### CODE REVIEW

**Review by fzipi — APPROVED** [review]
LGTM. Nice job!

File: waf.go [hunk 1]
```diff
@@ -107,3 +112,8 @@ func (w wafWrapper) NewTransaction() types.Transaction {
 func (w wafWrapper) NewTransactionWithID(id string) types.Transaction {
 	return w.waf.NewTransactionWithID(id)
 }
+
+// ToConfig returns a config copy of the WAF instance
+func (w wafWrapper) ToConfig() WAFConfig {
```
[the remark this answers is not in the document]
Comment (jcchavezs) [reply waf.go:117]: Ping @jptosso

**Comment by jcchavezs** [comment]
Is this still needed @jptosso ?

**Comment by jptosso** [comment]
Might be required when we come back to nginx

**Comment by jcchavezs** [comment]
LEt's close it and open back if needed.

**Pull request closed without merging**